src/styles/index.scss

```diff
@@ -1,3 +1,167 @@
+$buy-button-color: #00ACDC;
+$gray-text-color: #616070;
+$black-text-color: #060B35;
+
 body {
   margin: 0;
 }
+
+p {
+  margin: 0;
+}
+
+.product-card {
```

Comment:
move to `card.scss` file

Reply:
leave them in /styles folder or create /styles/blocks?